### PR TITLE
Ignore code parts

### PR DESCRIPTION
**Pull request by g4s8**
#523 - ignore code parts regex in `Par`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#785](https://github.com/zerocracy/farm/issues/785) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#785](https://github.com/zerocracy/farm/issues/785) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/681824140). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/test/resources/com/zerocracy/bundles/notify_awards/_after.groovy [hunk 1]
```diff
@@ -37,6 +37,6 @@ def exec(Project project, XML xml) {
   String comment = new Comment.Smart(issue.comments().get(1)).body()
   MatcherAssert.assertThat(
     comment,
-    Matchers.startsWith('Test reason: +15 points just awarded to ')
+    Matchers.startsWith('Test reason: +15 point(s) just awarded to ')
```
Comment (carlosmiranda) [inline src/test/resources/com/zerocracy/bundles/notify_awards/_after.groovy]: @g4s8 doesn't look like this test has anything to do with the change. Correct me if I am wrong

Comment (g4s8) [reply src/test/resources/com/zerocracy/bundles/notify_awards/_after.groovy]: @carlosmiranda looks like it didn't work and was fixed on 109 line in `Par`

Comment (carlosmiranda) [reply src/test/resources/com/zerocracy/bundles/notify_awards/_after.groovy]: @g4s8 right, that makes sense.

**Comment by carlosmiranda** [comment]
@yegor256 good to merge

**Comment by g4s8** [comment]
@yegor256 ping

**Comment by g4s8** [comment]
@yegor256 ping

**Comment by yegor256** [comment]
@rultor mege

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#785](https://github.com/zerocracy/farm/issues/785) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

No issues were found during code review. 
Please confirm that you'll try to find at least three problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda Thank you!

**Comment by ypshenychka** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**